src/main/kotlin/apply/application/ApplicantService.kt

```diff
@@ -48,17 +46,18 @@ class ApplicantService(
         applicantRepository.findByEmail(email) ?: throw IllegalArgumentException("email=$email 인 유저가 존재하지 않습니다")
 
     fun generateToken(applicantInformation: ApplicantInformation): String {
-        val applicant = applicantRepository.findByEmail(applicantInformation.email)
-            ?.also { it.validate(applicantInformation) }
-            ?: applicantRepository.save(applicantInformation.toEntity())
+        val encrypted = applicantInformation.copy(password = sha256Encrypt(applicantInformation.password))
+        val applicant = applicantRepository.findByEmail(encrypted.email)
+            ?.also { it.validate(encrypted) }
+            ?: applicantRepository.save(encrypted.toEntity())
 
         return jwtTokenProvider.createToken(applicant.email)
     }
 
-    fun changePassword(applicantId: Long, password: String) {
-        val applicant =
-            applicantRepository.findByIdOrNull(applicantId) ?: throw IllegalArgumentException("존재하지 않는 사용자입니다.")
-        applicant.password = password
+    fun changePassword(applicantId: Long, newPassword: String) {
+        applicantRepository.findByIdOrNull(applicantId)?.run {
+            password = sha256Encrypt(newPassword)
+        } ?: throw IllegalArgumentException("존재하지 않는 사용자입니다.")
```

Comment:
```suggestion
        applicantRepository.findByIdOrNull(applicantId)
        ?.run { password = sha256Encrypt(newPassword) } 
        ?: throw IllegalArgumentException("존재하지 않는 사용자입니다.")
```

위의 `generateToken`과 컨벤션을 맞춘다면 이렇게 수정할 수 있겠네요🙂